com.github.IsmaelMartinez.teams_for_linux.yml

```diff
@@ -1,15 +1,16 @@
 app-id: com.github.IsmaelMartinez.teams_for_linux
 base: org.electronjs.Electron2.BaseApp
-base-version: '22.08'
+base-version: '23.08'
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '23.08'
 sdk: org.freedesktop.Sdk
 command: teams-for-linux
 separate-locales: false
 finish-args:
   - --share=ipc
   - --socket=wayland
   - --socket=x11
+  - --socket=fallback-x11
```

Comment:
This should replace `--socket=x11` as in #90 :)

Reply:
I think this one is for runtime `23.08`. Replaced it now. Let's see. Thanks!